### PR TITLE
Setup database node

### PR DESCRIPTION
**Pull request by evmiguel**
This PR lays the foundation for running database migrations in Node.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "threeflows-v2",
```
Comment (gnarf) [inline package.json]: "threeflows" ?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "version": "0.0.1-prealpha.0",
```
Comment (gnarf) [inline package.json]: "2.0.0-prealpha.0" ?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "Redesign of TSL threeflows app",
```
Comment (gnarf) [inline package.json]: copy from the current one or try to write a better one actually describing the app?

**Comment by ktbee** [comment]
Looking good initially, but I think there is a small thing that needs fixing @evmiguel. When I run `npm run db-init-local`, I get the error:

`sh: db-migrate: command not found`

Does db-migrate need to be saved to the package.json? I didn't notice any dependencies there, so you might have it installed locally and just need to save it.

**Comment by ktbee** [comment]
Thanks for the package.json updates @evmiguel! I've gotten a little farther and have hit some different errors, let me know if I'm missing something from the steps below:

When I run `npm run db-init-local`, I get the following errors:

`psql: FATAL:  database "<my computer's name>" does not exist`
This error doesn't have to be a blocker for this PR, but it would be good to create a follow up issue to prevent it from happening for a better dev experience.

Then I get this error: 
```
threeflows@2.0.0-prealpha.0 db-create-role-dev: psql -c "CREATE ROLE tm WITH PASSWORD 'teachermoments'"; psql -c "ALTER ROLE tm WITH LOGIN"
npm ERR! Exit status 2
``` 
which does seem to be a blocker since our database user fails to be created.

Then "teachermoments" database is successfully created, though I get `psql: FATAL:  database  "<my computer's name>" does not exist` afterwards. This looks related to the first issue I mentioned above.

Next when I run `npm run db-migrate-up`, I get the error `[ERROR] AssertionError [ERR_ASSERTION]: ifError got unwanted exception: role "tm" does not exist`
This seems to prevent applying database migrations and was a blocker for me to continue with the rest of the instructions.

**Pull request merged**